kuma/users/utils.py

```diff
@@ -51,7 +51,9 @@ def retrieve_stripe_subscription_info(user):
         elif source.object == "source":
             card = source.card
         else:
-            return None
+            raise ValueError(
+                f"unexpected stripe customer default_source of type {source.object}"
```

Comment:
```suggestion
                f"unexpected stripe customer default_source of type {source.object!r}"
```

Reply:
Also, it says "type ..." but we're printing the *value*.  DOes that matter? Probably not.

Reply:
we're printing the value of `object` which specifies the type of `source`, so I'd argue it's correct